packages/api/src/schema/mutation.ts

```diff
@@ -884,6 +884,7 @@ export default [
       Remove user W3C push subscription object
       """
       removePushSubscription(p256dh: String!): User!
+      deleteAccount(userId: ID): Boolean!
```

Comment:
I don't like the naming "account" because we haven't used account across the whole system yet for API's. what about: 
- deleteUser ?